index.js

```diff
@@ -201,7 +220,60 @@ walletRouter.post('/send-tx', (req, res) => {
   const wallet = req.wallet;
   const outputs = req.body.outputs;
   const inputs = req.body.inputs || [];
-  const ret = wallet.sendManyOutputsTransaction(outputs, inputs)
+  // Expects object with {'uid', 'name', 'symbol'}
+  const token = req.body.token || null;
```

Comment:
Same here. `tokenId` instead of `token`.

Reply:
Resolved in the other comment